Styles.css

```diff
@@ -0,0 +1,342 @@
+* {
+  margin: 0;
+  padding: 0;
+  border-style: border-box;
+  font-family: "Roboto", sans-serif;
+}
+
+nav {
+  margin-top: 0.7em;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  height: 10vh;
+}
+
+.brand-img {
+  height: 52px;
+  width: 48px;
+  margin-right: 10em;
+  cursor: pointer;
+}
+
+.nav-right {
+  margin-left: 17em;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav-text {
+  font-weight: 100;
+  margin-right: 1.4em;
+  font-size: smaller;
+}
+
+.nav-text:hover {
+  cursor: pointer;
+  color: purple;
+}
+
+.nav-button {
+  font-size: 0.8em;
+  border: none;
+  padding: 0.6rem;
+  border-radius: 36px;
+  background-color: white;
+  border: 2px solid purple;
+  width: 100px;
+  color: purple;
+  cursor: pointer;
+}
+
+.nav-button:hover {
+  background-color: purple;
+  color: white;
+}
+
+h1 {
+  font-weight: 700;
+  text-align: center;
+  margin: 2em auto 0;
+  width: 720px;
+  display: inline-block;
+  font-size: 2.4em;
+}
+
+.Introduction {
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.Introduction__p {
+  text-align: center;
+  display: block;
+  font-weight: 200;
+  margin-top: 1.25em;
+  font-size: 1.15em;
+}
+
+.Introduction__avatar {
+  margin-top: 2em;
+  width: 170px;
+  display: inline-block;
+}
+
+.Introduction__heroMachine {
+  width: 520px;
+  margin-top: 3.7em;
+}
+
+.personal-info-container {
+  background-color: #7510f7;
+  display: flex;
+  height: 500px;
+  align-items: center;
+  flex-direction: column;
+  position: relative;
+}
+
+.personal-info-container__header {
+  text-align: center;
+  margin-top: 100px;
+}
+
+h2 {
+  color: white;
+  font-family: "Eurostile Extended", sans-serif;
+  font-size: 1.3em;
+}
+
+.personal-info-container__introduction {
+  padding-top: 1.5em;
+  color: white;
+  width: 620px;
+  text-align: center;
+  display: inline-block;
+  line-height: 1.7em;
+  font-weight: 100;
+  font-family: "Roboto", sans-serif;
+}
+
+.card-info {
+  display: grid;
+  grid-template-columns: repeat(1, 400px);
+  justify-content: center;
+  /* margin-top: -8em; */
+  position: absolute;
```

Comment:
dont use position absolute
when you change this to flex you wont need it anymore